go/minterm/minterm.go

```diff
@@ -42,7 +42,7 @@ func (m *MinTerm) Shutdown() error {
 	if m.termIn != nil {
 		go m.termIn.Close()
 	}
-	if m.termOut != nil {
+	if m.termOut != nil && m.closeTermOut {
```

Comment:
Ah sorry I missed this the first time through. Why was this working before? Or was it silently risking a close-someone-else's-file condition?